packages/prop-house-webapp/src/components/ReadMore/index.tsx

```diff
@@ -1,19 +1,36 @@
 import classes from './ReadMore.module.css';
-import { useState } from 'react';
+import { useLayoutEffect, useRef, useState } from 'react';
 
 const ReadMore: React.FC<{
   description: JSX.Element;
 }> = props => {
   const { description } = props;
 
-  const [readMore, setReadMore] = useState(false);
+  const ref = useRef(null);
+  const [open, setOpen] = useState(false);
+  const [showLink, setShowLink] = useState(false);
+
+  useLayoutEffect(() => {
+    // @ts-ignore
+    if (ref.current && ref.current.clientHeight < ref.current.scrollHeight) {
+      setShowLink(true)
+    }
+  }, [ref])
+
+  let classNames = classes.readMoreText;
```

Comment:
nit:
```
import clsx from 'clsx';
...
const classNames = clsx(classes.readMoreText, open && classes.clampLine);
```

Reply:
went ahead and just in-lined

`<div className={clsx(classes.readMoreText, open && classes.clampLine)} /> `